feature-libs/user/profile/components/register/register.component.html

```diff
@@ -184,6 +184,11 @@
               </label>
             </div>
           </div>
+          <cx-captcha
```

Comment:
Can we transform this component to one with form control embedded? See `DatePickerComponent` for an example.